guides/main_concepts/search.md

```diff
@@ -1,47 +1,87 @@
 # Search
 
-## Finding documents
+MeiliSearch has been designed to improve your search experience.
 
-When the query input is received, MeiliSearch is building a more complex query taking into account _typos_, n-grams, and _synonyms_ if configured.
+Today, finding a high-quality search engine may be complicated. Because we realized there was a lack of both powerful and simple solutions, we decided to create MeiliSearch. The **simplicity of use** is our philosophy and it keeps driving the development of MeiliSearch since the beginning of our adventure.
```

Comment:
I removed the part about `since the beginning of our adventure` because it feels to much like a blog post with that sentence.
```suggestion
Today, finding a high-quality search engine may be complicated. Because we realized there was a lack of both powerful and simple solutions, we decided to create MeiliSearch. The **simplicity of use** is our philosophy and it keeps driving the development of MeiliSearch.
```

Reply:
You're right! Thanks for the suggestion.